mimic/rest/maas_api.py

```diff
@@ -118,7 +118,7 @@ def create_entity(params):
     """
     params = collections.defaultdict(lambda: '', params)
     newentity = {}
-    newentity['label'] = params[u'label'].encode("utf-8")
+    newentity['label'] = params['label']
```

Comment:
We are not currently generating docs from it, but we really should be.  If you want to file a ticket feel free to assign it to me :).